concordium-sdk/src/main/java/com/concordium/sdk/responses/birkparamsters/Baker.java

```diff
@@ -0,0 +1,31 @@
+package com.concordium.sdk.responses.birkparamsters;
+
+import com.concordium.sdk.transactions.AccountAddress;
+import com.concordium.sdk.types.UInt64;
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * State of an individual baker.
+ */
+@Data
+@Getter
+@ToString
+public class Baker {
```

Comment:
Please make fields final. 
I don't see why `@ToString` and `@Getter` is necessary now you are using `@Data`.

Reply:
done